Codegen/test/resources/input/TypeCobol/CopyReplace4Colon/CopyReplace4Colon7.rdz.cbl

```diff
@@ -0,0 +1,13 @@
+       IDENTIFICATION division.
```

Comment:
We also need test on the parser project. Can you move this one to TypeCobol.Test project ?

Reply:
ok, I will

Reply:
done by 
1434b65